src/core/evaluator.js

```diff
@@ -1129,6 +1130,7 @@ class PartialEvaluator {
         case "Font":
           isSimpleGState = false;
 
+          // eslint-disable-next-line arrow-body-style
```

Comment:
We could probably do something like
```js
promise = promise.then(() => this.handleSetFont....)
                               .then(loadedName = > { ... });
```

Reply:
This is one of the cases where, as mentioned in the commit message, I don't feel that it's really a readability improvement to change that. (Since there's *multiple* Promises involved here.)